src/main/java/edu/itu/cavabunga/caldav/rest/CalendarRest.java

```diff
@@ -0,0 +1,142 @@
+package edu.itu.cavabunga.caldav.rest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CalendarRest {
```

Comment:
Actually, I did not understand what is the difference between rest and normal component, is it a duplicated code? At least Calendar should be an abstract class in my POV.